server/src/style/javaHeader.txt

```diff
@@ -1,14 +1,14 @@
-/*
- * Copyright (c) 2020 Broadcom.
- * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
- *
- * This program and the accompanying materials are made
- * available under the terms of the Eclipse Public License 2.0
- * which is available at https://www.eclipse.org/legal/epl-2.0/
- *
- * SPDX-License-Identifier: EPL-2.0
- *
- * Contributors:
- *    Broadcom, Inc. - initial API and implementation
- *
- */
+^/\*$
+^ \* Copyright \(c\) \d\d\d\d Broadcom\.$
```

Comment:
I think better to keep 2020 as for validation check. According to [US](https://rally1.rallydev.com/#/377806736628/teamboard?detail=%2Fuserstory%2F360326325276&view=1a5c8a9d-edbf-45aa-818c-215d6aa8f7af) we need to keep 2020 from now.

Reply:
This link is private. This decision should be available for people outside Broadcom.

Reply:
Easy, I will create a GitHub issue with info ;)

Reply:
d\d\d\d\  allows any 4 digit number - I guess we do need to manual check if year is actual (2021, ... ) right?

Reply:
Yes, it allows any 4 digit number.
We can do something like `202\d`, but we have to change this in the future. With \d\d\d\d we are safe up to eight centuries =)

Reply:
You are too much optimistic about the future of this project

Reply:
He is right to be so. It's the mainframe world.